usr/local/bin/um-setup

```diff
@@ -76,23 +164,130 @@ rm -rf $tmpdir
 
 echo "GCOM library has been installed to $HOME/umdir/gcom/default"
 
+##### Install shumlib #####
+
+tmpdir=$(mktemp -d)
+if [ ! -d "$tmpdir" ]; then
+  echo "Unable to create temporary directory"
+  exit 1
+fi
+
+echo
+echo "Finding Shumlib source..."
+get_source_tree "$shumlib_src" "$tmpdir"     # Sets global $vn
+
+echo "Installing Shumlib from: $shumlib_src"
+
+mkdir -p $HOME/umdir/shumlib
+
+cd $tmpdir/wc
+vm_config='vm-x86-gfortran-gcc'
+echo "Building Shumlib dependencies"
+
+# shumlib is built twice, with and without OpenMP:
+for shum_omp in true false; do
+
+  if $shum_omp; then
+    omp=openmp
+  else
+    omp=no-openmp
+  fi
+
+  install_dir="$HOME/umdir/shumlib/$vn/$vm_config/$omp"
+  mkdir -p "$install_dir"
+
+
+  # Remove any previous builds
+  make -f make/${vm_config}.mk clean > /dev/null 2>&1 
+
+  # Build the library
+  echo
+  echo "Building Shumlib $omp library..."
+  echo "Make output at $install_dir/make.log"
+  SHUM_OPENMP=$shum_omp make -f make/${vm_config}.mk shum_wgdos_packing &> "$install_dir/make.log"
```

Comment:
I'm probably missing something, but is this saying that if ``shum_omp=false`` then you want to run ``false make -f ....``? Should it be ``$omp make -f ..." instead? (I can't remember how that gets built...)